packages/support/src/Commands/Concerns/CanReadModelSchemas.php

```diff
@@ -43,21 +44,21 @@ protected function guessBelongsToRelationshipName(AbstractAsset $column, string
     {
         /** @var Model $modelInstance */
         $modelInstance = app($model);
-        $modelInstanceReflection = invade($modelInstance);
+        $modelInstanceReflection = new ReflectionClass(invade($modelInstance));
```

Comment:
Do we really need invade here?

```suggestion
        $modelInstanceReflection = new ReflectionClass($modelInstance);
```

Reply:
yes, you right.

Reply:
Did you test your previous code?

Reply:
```php
class Invader
{
    /** @var T */
    public object $obj;
    public ReflectionClass $reflected;

    /**
     * @param T $obj
     */
    public function __construct(object $obj)
    {
        $this->obj = $obj;
        $this->reflected = new ReflectionClass($obj);
    }
...
```

This was in version 1.1.1.

In 2.0 is this
```php
class Invader
{
    /**
     * @param T $obj
     */
    public function __construct(
        public object $obj
    ) {
    }
...
```